charts/core/ChartDimension.ts

```diff
@@ -220,7 +220,7 @@ export class ChartDimension {
     }
 
     @computed get sortedNumericValues(): number[] {
-        return sortBy(this.values.filter(isNumber).filter(v => !isNaN(v)))
+        return sortNumeric(this.values.filter(isNumber).filter(v => !isNaN(v)))
```

Comment:
Awesome! 120ms is definitely noticeable. Great this will improve all charts as well. Even if it saves only 20ms on smaller charts, 20ms x 3000 = 1 minute savings off an all chart bake!